h3_test.go

```diff
@@ -609,6 +609,7 @@ func TestPentagons(t *testing.T) {
 	t.Parallel()
 
 	for _, res := range []int{0, 8, 15} {
+		res := res
```

Comment:
Is this needed for anything?

Reply:
I added it because `go vet` was complaining about it with this error: `loop variable res captured by func literal `